docs/circuits.md

```diff
@@ -328,7 +328,7 @@ from an ``OP_TREE`` via the static method `Circuit.from_ops`
 (which takes an insertion strategy as a parameter):
```

Comment:
You missed a from_ops in this file. I can't comment on the line, but it's right above here.

Reply:
That's a pretty significant one; the example didn't match the description anymore